api/src/main/java/marquez/db/JobDao.java

```diff
@@ -386,4 +447,64 @@ JobRow upsertJob(
       String location,
       UUID symlinkTargetId,
       PGobject inputs);
+
+  @SqlUpdate(
+      """
+    WITH new_tag AS (
+    INSERT INTO tags (uuid, created_at, updated_at, name, description)
+    SELECT
+      gen_random_uuid(),
+      NOW(),
```

Comment:
We pass along the timestamp from the application layer to the data access layer ensuring the timezone of the timestamp is `UTC` (i.e. of the running HTTP API server).  We set the JVM timezone on startup here [`entrypoint.sh`](https://github.com/MarquezProject/marquez/blob/main/docker/entrypoint.sh#L16)

I would add a `now` param (see [example](https://github.com/MarquezProject/marquez/blob/main/api/src/main/java/marquez/db/TagDao.java#L26)):
```
updateJobTags(String namespaceName, String jobName, String tagName, Instant now)
```

Reply:
Cool that's done.